src/libs/rover/domain.cpp

```diff
@@ -49,9 +51,9 @@ Domain::set_render_settings(const RenderSettings &settings)
   {
 #endif
 
-  ROVER_INFO("Render mode = energy");
   auto engine = std::make_shared<EnergyEngine>();
-  engine->set_unit_scalar(settings.m_energy_settings.m_unit_scalar);
+  float64 unit_scalar = m_settings["rover/unit_scalar"].value();
+  engine->set_unit_scalar(unit_scalar);
```

Comment:
```suggestion
  engine->set_unit_scalar(m_settings["rover/unit_scalar"].value());
```

Reply:
also, set_unit_scalar takes a float32: https://github.com/Alpine-DAV/ascent/blob/ecd571649d5b5e320d8270cb850680167ec6a7cd/src/libs/rover/energy_engine.hpp#L41
So we are potentially doing a bunch of unnecessary conversions.